app/views/sephora_style_guide/home/_checkbox_radio.slim

```diff
@@ -0,0 +1,46 @@
+article#checkbox-radio
+  h2.h2 Checkboxes & Radio
+
+  section
+    h3.h3 Checkbox
+    .row
+      .col-2
+        .option-link @click='toggleCheck'
+          span v-if='isCheckboxChecked'
+            = inline_svg 'sephora_style_guide/checkbox-checked.svg', class: 'checkbox'
+          span v-if='!isCheckboxChecked'
+            = inline_svg 'sephora_style_guide/checkbox.svg', class: 'checkbox'
```

Comment:
I am actually re-considering the use of `inline_svg` now that we are using Vue... Will research about it and let you know. You could also do some research if you are free.